pkg/fftm/manager.go

```diff
@@ -87,6 +88,7 @@ type manager struct {
 	blockListenerDone       chan struct{}
 	started                 bool
 	apiServerDone           chan error
+	debugServer             *http.Server
```

Comment:
This change itself makes sense to me, but I was just curious as to the "why" behind it? Did it make it possible to debug something that wasn't possible before?

Reply:
You can now use the debug API to query go's pprof tooling. gather profiles to turn into flame graphs, etc.